pkg/pluginutil/pluginutil.go

```diff
@@ -45,6 +47,43 @@ func HTTPClient(baseURL string, opts ...httpclient.Option) *httpclient.Client {
 	return httpclient.New(strings.TrimRight(baseURL, "/"), append(baseOpts, opts...)...)
 }
 
+// NewHTTPClient returns an HTTP client from a plugin runtime.
+// It is created with the `client-go` package and will eventually
+// replace the httpClient of the CLI.
+func NewHTTPClient(baseURL string) *http.Client {
```

Comment:
Just noticed we don't set our CLI specific user agent when we use the httpclient from client-go in general, but I think it requires change in the package first.